app/views/amusement_parks/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1><%= @amusement_park.name %></h1>
+
+<p>Price: <%= @amusement_park.admission_cost %></p>
+
+<h2>Rides:</h2>
+<section id="rides">
+  <% @amusement_park.ride_avg_experience.each do |ride| %>
+    <p><%= ride.ride_name %>, Average experience of mechanics: <%= ride.avg_experience %> years</p>
+  <% end %>
+</section>
+
+<h2>Mechanics working on rides:</h2>
+<section id="mechanics">
+  <% @amusement_park.find_mechanics.each do |mechanic| %>
+    <p><%= mechanic %></p>
+  <% end %>
```

Comment:
I'm realizing that if you updated `find_mechanics` to the way I suggested it above, you'd need to call `mechanic.name` on line 15. 

You could, instead, tack on a `.pluck(:name)` to the query in the `find_mechanics` method, and then you could keep this logic the same. 

Either is fine by me!